services/pgnotifier/pgnotifier.go

```diff
@@ -334,7 +337,7 @@ func (notifier *PgNotifierT) Publish(jobs []JobPayload, priority int) (ch chan [
 	if err != nil {
 		return
 	}
-	defer stmt.Close()
+	defer func() { _ = stmt.Close() }()
```

Comment:
log as warn ?

Reply:
Might just be noise, the close happens after we potentially committed successfully. Who's going to look at that warning and what kind of action would you take if you saw one? It sounds more like pollution. WDYT?